beesdoo_shift_swap/wizard/exchange_wizard.py

```diff
@@ -54,14 +56,21 @@ def _get_template_dated(self):
     @api.onchange("worker_id")
     def _get_available_tmpl_dated(self):
```

Comment:
Maybe change this method name to prevent a confusion with `get_available_tmpl_dated`

Reply:
Done